geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RegionManagementController.java

```diff
@@ -196,4 +196,20 @@ public ClusterManagementResult deleteRegion(
     return new ResponseEntity<>(result,
         HttpStatus.CREATED);
   }
+
+  @ApiOperation(value = "delete region index")
+  @PreAuthorize("@securityService.authorize('CLUSTER', 'MANAGE', 'QUERY')")
+  @DeleteMapping(REGION_CONFIG_ENDPOINT + "/{regionName}" + INDEXES + "/{indexName:.+}")
```

Comment:
Did you consider adding a test to verify ":.+" is working to RegionManagementControllerSpringTest? Currently that is all that test does so you should be able to copy one of its current methods to cover this endpoint

Reply:
We did make a change to the DUnit test to do the same and that will exercise the endpoint. I can add one to the `RegionManagementControllerSpringTest` as well.